docs/diagrams/ShowEvents.puml

```diff
@@ -13,7 +13,7 @@ box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("showevents")
+[-> LogicManager : execute("listevents")
 activate LogicManager
 
```

Comment:
You forgot to change the parseCommand part to listevents I think

Reply:
Fixed